frame/executive/src/lib.rs

```diff
@@ -220,6 +220,58 @@ where
 		weight
 	}
 
+	/// Execute given block, but don't do any of the [`final_checks`].
+	///
+	/// Should only be used for testing.
+	#[cfg(feature = "try-runtime")]
+	pub fn execute_block_no_state_root_check(block: Block) -> frame_support::weights::Weight {
+		Self::initialize_block(block.header());
+		Self::initial_checks(&block);
+
+		let (header, extrinsics) = block.deconstruct();
+
+		Self::execute_extrinsics_with_book_keeping(extrinsics, *header.number());
+		// don't call `final_checks`, but do finalize the block.
+		let _header = <frame_system::Pallet<System>>::finalize();
+
+		frame_system::Pallet::<System>::block_weight().total()
+	}
+
+	// #[cfg(feature = "try-runtime")]
```

Comment:
this is my half finished attempt at disabling the signature verification in `try-runtime`. will either use, or remove later.